network/onchain-discovery/src/test.rs

```diff
@@ -195,24 +195,26 @@ fn gen_single_config(count: usize) -> (NodeConfig, ValidatorSet, DiscoverySet) {
 
 fn setup_storage_service_and_executor(
     config: &NodeConfig,
-) -> (Runtime, Arc<dyn StorageRead>, Executor<LibraVM>) {
+) -> (Runtime, Arc<dyn StorageRead>, Executor<LibraVM>, Waypoint) {
     let (arc_db, db_reader_writer) = init_libra_db(config);
     let genesis_tx = config.execution.genesis.as_ref().unwrap();
-    bootstrap_db_if_empty::<LibraVM>(&db_reader_writer, genesis_tx)
-        .expect("Db-bootstrapper should not fail.");
+    let waypoint = bootstrap_db_if_empty::<LibraVM>(&db_reader_writer, genesis_tx)
+        .expect("Db-bootstrapper should not fail.")
```

Comment:
what's the point of this expect statement?

Reply:
When it fails you get an error message?